tests/pr_test.go

```diff
@@ -1,25 +1,42 @@
-// Tests in this file are run in the PR pipeline
 package test
 
 import (
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/cloudinfo"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
-const defaultExampleTerraformDir = "examples/default"
+const defaultExampleTerraformDir = "examples/observability_default"
+const resourceGroup = "geretain-test-observability-instances"
+
+var sharedInfoSvc *cloudinfo.CloudInfoService
+
+// TestMain will be run before any parallel tests, used to set up a shared InfoService object to track region usage
+// for multiple tests
+func TestMain(m *testing.M) {
+	sharedInfoSvc, _ = cloudinfo.NewCloudInfoServiceFromEnv("TF_VAR_ibmcloud_api_key", cloudinfo.CloudInfoServiceOptions{})
+
+	os.Exit(m.Run())
+}
 
 func TestRunDefaultExample(t *testing.T) {
 	t.Parallel()
 
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  defaultExampleTerraformDir,
-		Prefix:        "mod-template",
-		ResourceGroup: resourceGroup,
+		Testing:                       t,
+		TerraformDir:                  defaultExampleTerraformDir,
+		Prefix:                        "obs-all",
+		ResourceGroup:                 resourceGroup,
+		CloudInfoService:              sharedInfoSvc,
+		ExcludeActivityTrackerRegions: true,
+		TerraformVars: map[string]interface{}{
+			"logdna_plan":           "7-day",
+			"sysdig_plan":           "graduated-tier",
+			"activity_tracker_plan": "7-day",
```

Comment:
I think we need to overwrite the settings for `enable_platform_metrics` and `enable_platform_logs` in the test and set them to false, since you can only have 1 default receiver per account

Reply:
Do I want to do it here or in the examples variables? I think these should also be moved to the examples variables